skimage/filters/tests/test_get_window.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from scipy.signal import get_window as get_window1d
+from skimage.filters import get_window
+from skimage._shared.testing import parametrize, raises
+
+
+@parametrize("window",
+             [16,
+              'hann',
+              ('tukey', 0.8)])
+@parametrize("size", [5, 8, 15])
+@parametrize("ndim", [2, 3, 4])
```

Comment:
how much time does it take to execute the test with `ndim=4` and `size=15`? The test suite is quite long, so we want to make sure a new test executes in a time way below the s (ideally below 0.1 s I would say).

Reply:
The tests I had in there were way overkill. Pruned way back